### PR TITLE
Update apps-deploy.md

### PR DESCRIPTION
**Pull request by storkersen**
Microsoft has moved the option to assign apps from Manage on the apps page to be inside the Property page.

### CODE REVIEW

**Comment by PRMerger14** [comment]
@storkersen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/apps/apps-deploy.md [hunk 1]
```diff
@@ -68,9 +68,10 @@ The following table lists the various options for *assigning* apps to users and
 1. Sign in to the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
 2. Select **Apps** > **All apps**.
 3. In the **Apps** pane, select the app you want to assign.
-4. In the **Manage** section of the menu, select **Assignments**.
-5. Select **Add Group** to open the **Add group** pane that is related to the app.
-6. For the specific app, select an **assignment type**:
+4. In the **Manage** section of the menu, select **Properties**.
+5. Scroll down to **Properties**, select **Assignments**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-deploy.md:72]:
```suggestion
5. Scroll down to **Properties** and select **Assignments**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @storkersen, kindly commit the minor suggested change. Many thanks.

**Comment by PRMerger9** [comment]
@storkersen : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and signoff this PR. Thanks

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**